setup/security/authorization/pipeline-permissions/index.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: single
+title:  "Pipeline Permissions"
+sidebar:
+  nav: setup
+---
+
+{% include alpha version="1.9" %}
+
+{% include toc %}
+
+Pipeline permissions enable automatically triggered pipelines to modify
+resources in protected accounts and applications. They are an alternative
+to manually managing Fiat Service Accounts.
```

Comment:
Maybe link here to /setup/security/authorization/#service-accounts/ or to /setup/security/authorization/service-accounts/

Reply:
Done

Reply:
Have you tested the link? I don't think we use relative paths anywhere else in our internal links. I could be wrong.

Reply:
yes, it worked locally.